lua/packer.lua

```diff
@@ -519,6 +550,24 @@ packer.loader_complete = function(lead, _, _)
   return completion_list
 end
 
+-- Snapshot installed plugins
+-- Given a list of plugins it will only snapshot said plugins,
+-- otherwise it will snapshot all plugins (except for disabled)
+packer.snapshot = function (filename)
+  async(function()
+    --local start_time = vim.fn.reltime()
+    log.info(string.format("Taking snapshots of currently installed plugins to %s...", filename))
+    filename = util.join_paths(config.snapshot_path, filename)
+    await(snapshot(filename, plugins))
+    await(a.main)
+    --local delta = string.gsub(vim.fn.reltimestr(vim.fn.reltime(start_time)), ' ', '')
+    --print(string.format("delta: %s", delta))
+    log.info("Snapshot complete")
+    packer.on_complete()
```

Comment:
Need to document that `on_complete` triggers for snapshots too (not just installs, etc.)